### ISSUE TITLE
Create a new RDS data source where you can get data of a specific RDS

### ISSUE THREAD

**Issue by mele78** [opentelekomcloud/terraform-provider-opentelekomcloud#1736]
# Description of the enhancement
There is no RDS data source where you can read values from an existing RDS. So a new data source should be created.
## Example
```
data "opentelekomcloud_rds_instance_v3" "rds" {
  name = "rds-instance-01"
}
```
The best case would be to get all the known values which are used by this specific RDS. Idea is to use the attributes from opentelekomcloud_rds_instance_v3 resource.
## Usecase
The specific usecase was to get the private IP of an RDS to add it inside another project. This was not possible, could only be solved by terraform_remote_state data source. This is no optimal solution.
# Provider Version
1.29.2
# Affected Resource(s)
New resource needed

**Comment by vladimirvshivkov** [comment]
Hello, @mele78!
Thanks for the issue. This is definetely data source we currently missing.
Will work on it and We will add it in the next releases.

**Comment by vladimirvshivkov** [comment]
Closed via #1739
